src/styles/index.scss

```diff
@@ -1,3 +1,123 @@
+* {
+  margin: 0;
+}
```

Comment:
don't zero your margins under this selector, it's inefficient

Reply:
not fixed
we need to reset the styles of only those components where we need it:
```
a, p {   // etc.
  margin: 0
} 
```